### DISCUSSION TITLE
Capacitor 3.0.0-rc.0 Ionic Appflow Integration Error

### DISCUSSION THREAD

**Discussion by chetanbansal019** [ionic-team/capacitor#4387] [Q&A]
Hi Everyone,

I have been trying to integrate Appflow into a new Capacitor app which is using version 3.0.0-rc.0. I am the following error on this particular version of capacitor.
Appflow is successfully working on an older Capacitor app based on version 2.4.2.
Here is the error that I see
![image](https://user-images.githubusercontent.com/28653223/112041396-5db7fd80-8b6c-11eb-9aeb-aacb8510dec1.png)
![image](https://user-images.githubusercontent.com/28653223/112041438-6b6d8300-8b6c-11eb-95b9-5ce464a2c0e6.png)

I am not sure if this is a bug or some misconfiguration from my end. 
Let me know if any more details about my setup are required.

Here is the test code repository where the issue can be reproduced.
[Repository](https://github.com/chetanbansal019/testappflow)

**Comment by Sampath-Lokuge** [comment]
You need to use `CLI 3.0.0-rc.0` too.

**Reply by chetanbansal019** [reply]
In my thinking, the CLI version mismatch will not impact this.. Anyways, upgraded the CLI but the error is still there..

**Reply by Sampath-Lokuge** [reply]
Hmm.. You can open an Appflow support ticket and they'll give feedback within 24 H.

**Reply by chetanbansal019** [reply]
oh yes, let me do that.. Thank you

**Comment by jcesarmobile** [comment]
Did you add the `import '@capacitor/core';` in your app?
i.e. in Angular apps on top of `src/main.ts` file
in React apps on top of `src/index.tsx` file

**Reply by chetanbansal019** [reply]
I added it after your suggestion.. but still the same issue..
seems very inconsistent too..
check the last line in this screenshot..
![image](https://user-images.githubusercontent.com/28653223/112200746-8a841780-8c35-11eb-8cfe-96116db213f4.png)

here is another log from xcode console, if this might help.
![image](https://user-images.githubusercontent.com/28653223/112200938-c1f2c400-8c35-11eb-90e7-e72470151bf8.png)

**Reply by chetanbansal019** [reply]
I have added the sample app where the issue can be reproduced.

**Reply by jcesarmobile** [reply]
The `import '@capacitor/core';` should be enough to make it work on iOS, but it won't work on Android.
We are working on a fix that won't require adding the import and should work on both iOS and Android.

**Reply by chetanbansal019** [reply]
but this error is there for both platforms.. adding the import statement didn't help.. I checked for v2.4.2, which works perfectly, the console log shows this

![image](https://user-images.githubusercontent.com/28653223/112520184-2aba7780-8dc1-11eb-9cf1-c5dffef058a2.png)

**Reply by jcesarmobile** [reply]
update to rc.1, should work fine now
